.stylelintrc

```diff
@@ -0,0 +1,11 @@
+{
+  "processors": ["stylelint-processor-styled-components"],
+  "extends": "stylelint-config-standard",
+  "syntax": "scss",
+  "rules": {
+    "property-no-unknown": [true, { "ignoreProperties": ["resize-mode"] }],
+    "block-opening-brace-space-after": null,
+    "block-closing-brace-space-before": null,
+    "declaration-block-no-redundant-longhand-properties": null
```

Comment:
RN supports different rules for CSS occasionally, following this linter will get unexpected fails